content/blog/okta-sso-integration-for-green-lake-and-aruba-central.md

```diff
@@ -0,0 +1,150 @@
+---
+title: Configuring SSO for Aruba Central and HPE GreenLake using Okta
+date: 2023-02-14T20:11:58.846Z
+externalLink: ""
+author: "Will Smith - Consulting Systems Engineer (ACEX #34)"
+authorimage: /img/willsmith-sm.jpg
+thumbnailimage: /img/gl-sso.jpg
+disable: false
+tags:
+  - aruba-central
+  - hpe-greenlake-cloud-platform
+  - sso
+  - okta
+---
+Aruba Central has gone GREEN…GreenLake that is! If you’re reading this, then you’ve likely heard that Aruba Central is now integrated into the new HPE GreenLake Common Cloud Platform (GLCP). This allows IT admins to view and orchestrate critical network services such as Wired, Wireless and SD-Branch, through the same dashboard as their compute and storage infrastructure.
+
+If you are new to Aruba Central and are looking to enable SSO, this guide is for you. This will walk you through the process of configuring SSO for GreenLake and Aruba Central using Okta.
```

Comment:
Updated: 
If you are new to Aruba Central and are looking to enable SSO, this guide is for you. It will walk you through the process of configuring SSO for HPE GreenLake and Aruba Central using Okta.